src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -44,7 +44,7 @@ def check_status
         # raise BadStatus, propagating the metadata if present.
         md = status.metadata
         with_sym_keys = Hash[md.each_pair.collect { |x, y| [x.to_sym, y] }]
```

Comment:
A value in the hash can be a list of strings, which is handled appropriately in the core.